docs/get-started/getting-started-workload-qualification.md

```diff
@@ -101,15 +101,11 @@ the driver logs with `spark.rapids.sql.explain=all`.
    [configuration documentation](../configs.md) for details of RAPIDS Accelerator
    parameters.
 
-   For example, if your jobs have `double`, `float` and `decimal` operators together with some Scala
+   For example, if your jobs have `decimal` operators together with some Scala
    UDFs, you can set the following parameters:
 
   ```scala
-   spark.conf.set("spark.rapids.sql.incompatibleOps.enabled", true)
-   spark.conf.set("spark.rapids.sql.variableFloatAgg.enabled", true)
    spark.conf.set("spark.rapids.sql.decimalType.enabled", true)
```

Comment:
I believe removed spark.rapids.sql.decimalType.enabled in https://github.com/NVIDIA/spark-rapids/commit/de6d45492082ab107829cb810bb2312db7744c40#diff-12777fe3bbb26d1a892f120acf176b07af222570dafd19f039adc3f84d4c58e2L547 , and should remove it from this doc. 
```suggestion
```